package chapter14.exception;

public class ArrayExceptionHandling {
	public static void main(String[] args) {
		int[] arr = new int[5]; // 힙 영역에 int형 기억장소 5개 생성.
		
		// arr[0] ~ arr[4] 힙 영역에 생성된 기억장소 요소를 가리키는 이름
		
		// Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException 오류 발생
		// arr[5] = 10; 
		
		for(int i = 0; i < 5; i++) {
			System.out.println(arr[i]);
		}
		
		System.out.println("테스트");
	}
}
